.gitignore

```diff
@@ -13,3 +13,4 @@
 *.a
 mkmf.log
 log/*.log
+.ruby-version
```

Comment:
Its not sth you should keep it project .gitignore file, but your global one: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore because its related to your local setup